shellpython/shellpy.py

```diff
@@ -3,20 +3,21 @@
 import os
 import subprocess
 from shellpython.preprocessor import preprocess_file
+from argparse import ArgumentParser
+import re
 
 
 def main():
-    if len(sys.argv) == 1 or sys.argv[1] in ['-h', '--help', 'help']:
-        print('Shellpy, write shell scripts in Python easily')
-        print('Usage: shellpy [*.spy script] [arguments]')
-        exit(0)
+    def shellpyFile(fil):
```

Comment:
I think we cannot put any constraint to user as he does not know which parameter for shellpy will be introduced tomorrow. So user should be free to use any names to params and should be free to adverts parameters as he wants, either by index argv[1] or by name like in argparse.